### PR TITLE
Mental wellness

### PR DESCRIPTION
**Pull request by mimif10**
included mental features and pages

### CODE REVIEW

File: app/src/main/java/com/example/life_is_well/MentalPageMain.kt [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
+    private lateinit var binding: ActivityMentalPageMainBinding
+
+    override fun onCreate(savedInstanceState: Bundle?)
+    {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_mental_page_main)
+
+        binding = ActivityMentalPageMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        // setContentView(R.layout.activity_mental_page_main)
```
Comment (jacobburgo) [inline app/src/main/java/com/example/life_is_well/MentalPageMain.kt:18]: Still needed?

File: app/src/main/res/values-night/strings.xml [hunk 2]
```diff
@@ -6,8 +6,42 @@
...
+    <string name="title_notifications">Notifications</string>
+    <string name="title_activity_breathing_exercixe">BreathingExercixe</string>
+    <string name="title_activity_breathing_exercise_mental_page">BreathingExerciseMentalPage</string>
+    <string name="defaultNavHost">nav_host</string>
+    <string name="navGraph">nav_graph</string>
+    <string name="nameofuser">Name</string>
+    <string name="account">Account 1</string>
+    <string name="new_account">New Account</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (jacobburgo) [inline app/src/main/res/values-night/strings.xml:38]: Just making note for me and anyone else that this lil section can be removed at any point along the road.

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <dimen name="fab_margin">16dp</dimen>
+    <!-- Default screen margins, per the Android Design guidelines. -->
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
+</resources>
```
Comment (jacobburgo) [inline app/src/main/res/values/dimens.xml:6]: 👍🏼 🔥

File: local.properties [hunk 4]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Wed Sep 07 10:00:38 EDT 2022
+sdk.dir=C\:\\Users\\mamin\\AppData\\Local\\Android\\Sdk
```
Comment (jacobburgo) [inline local.properties]: Noting this for everyone that we will have to address this file each after this merges in. Don't worry, no problems. Once my PR #21 gets merged in this file will be removed from the repo and exist only locally for us all. With that being that case. We should merge this PR, then PR #21, then PR #22 .

**Review by jacobburgo — COMMENTED** [review]
LGTM :shipit:

**Review by jacobburgo — APPROVED** [review]
Meant to approve.

**Review by MillerShane — APPROVED** [review]
Looks good

**Pull request merged**